backend/src/LibExecution/Dval.fs

```diff
@@ -104,3 +104,18 @@ let result
   match dv with
   | Ok dv -> resultOk okType errorType dv
   | Error dv -> resultError okType errorType dv
+
+
+let byteArrayToDvalList (bytes : byte[]) : Dval =
```

Comment:
Not sure if this is the right place for these two helper functions

Reply:
This is where I would put them too